app/src/main/java/co/nimblehq/compose/crypto/ui/theme/Theme.kt

```diff
@@ -33,6 +33,16 @@ fun ComposeTheme(
         Palette.ComposeLightPalette
     }
 
+    val systemUiController = rememberSystemUiController()
+    DisposableEffect(systemUiController, darkTheme) {
```

Comment:
@luongvo Regarding [this](https://github.com/nimblehq/jetpack-compose-crypto/pull/30#discussion_r965736522) I don't think we need `rememberSystemUiController` as we can use `systemBarsPadding()` from `androidx.compose.foundation`, Could you please following these steps.
1. Comment all changes in `Theme.kt`.
2. In `MainActivity` add `WindowCompat.setDecorFitsSystemWindows(window, false)` inside `onCreate` method and put it above `setCotent` scope.
3. Upgrade `COMPOSE_VERSION` to `1.2.0-rc01` (You need to upgrade `ANDROID_COMPILE_SDK_VERSION` -> `32` and `KOTLIN_VERSION` to `1.6.21`)
4. In `HomeScreen` In line `29` add `modifier = Modifier.fillMaxSize().systemBarsPadding()`
5. In `style.xml` comment these item:
   
<img width="706" alt="Screen Shot 2565-09-08 at 17 20 06" src="https://user-images.githubusercontent.com/28002315/189098296-6d02030b-38d1-432f-939c-79c99744c98e.png">
6. Run and test

Reply:
@Wadeewee Okay, 
- using `WindowCompat.setDecorFitsSystemWindows(window, false)` and `Modifier.fillMaxSize().systemBarsPadding()` works for me ✅ 
- We will need to use `rememberSystemUiController` if we need to switch between full and non-full screen content in each screen later.
- we still need to keep `<item name="android:windowBackground">@color/colorBackground</item>` to set the correct background at app starting up.
- we still need to keep `<item name="android:navigationBarColor">@android:color/transparent</item>` to make the full immersive effect for both top status bar and bottom navigation bar too.

Reply:
@Wadeewee Fixed in https://github.com/nimblehq/jetpack-compose-crypto/pull/30/commits/0c761c0af38631e1bdd7babc5aca8a34c1482398

Reply:
>- We will need to use rememberSystemUiController if we need to switch between immersive and non-immersive >screen content in each screen later.
>- we still need to keep <item name="android:windowBackground">@color/colorBackground</item> to set the >correct background at app starting up.
>- we still need to keep <item name="android:navigationBarColor">@android:color/transparent</item> to make the >full immersive effect for both top status bar and bottom navigation bar too.

@luongvo I'm not sure about these 🙏 So Let's keep it then, I will recheck after this PR was merged and test on this [PR](https://github.com/nimblehq/jetpack-compose-crypto/pull/28) again, If we need it🚀